aries-site/src/data/wcag/datafilters.json

```diff
@@ -0,0 +1,512 @@
+[
+  {
+    "rule": "1.1.1",
+    "label": "Non-text content",
+    "status": "conditional",
+    "notes": "if using Layer for datafilters then Layer with a close icon must provide an aria-label or alternative text unless the icon is purely decorative"
```

Comment:
I don't think we provide a way to pass an aria-label to the close button

Reply:
I would think we'd provide that. The aria-label for the X is "Close". Does it need more detail like "Close filters"?

Reply:
^ Yes I think we should provide a default and people can override it via messages if necessary

Reply:
yes we have a default but do not have a way to override.

Reply:
@jcfilben would you consider this is still conditional? or passed

Reply:
Let's fix to give more detail and then after it is fixed mark as passed. Right now not sure the best way to mark this since the current aria-label isn't very descriptive